Backend/internal/cmsjson/marshall.go

```diff
@@ -107,13 +107,14 @@ func (c Configuration) marshallPrimitive(field reflect.Value) []byte {
 func (c Configuration) marshallCore(source reflect.Value) []byte {
```

Comment:
Maybe use switch statements for the function above this (`marshallPrimitive`)?

Reply:
I can't seem to leave a comment on that piece of code since the code wasn't changed in this PR.